helpers.go

```diff
@@ -0,0 +1,21 @@
+package realis
+
+import (
+	"context"
+
+	"github.com/aurora-scheduler/gorealis/v2/gen-go/apache/aurora"
+)
+
+func (r *Client) jobExists(key aurora.JobKey) (bool, error) {
+	resp, err := r.client.GetConfigSummary(context.TODO(), &key)
+	if err != nil {
+		return false, err
+	}
+
+	return !(resp == nil ||
```

Comment:
we have to be very careful with.
Can you make a test and verify this function?
it would be great if we have unit test for this function.

Reply:
Created a unit test for the method.
Surprised to find out that the method written was never worked. The one in V1 always returns false. When I added a negation, it always returns true.
Now it should work

Reply:
I think we should fix them like
a != nil || a.B !=nil instead of !(a == nil || a.B == nil)

Reply:
Do you mean a!= nil && a.B!= nil && a.C!=nil ?

Reply:
Done